pkg/transform/kubernetes/kubernetes.go

```diff
@@ -289,6 +289,53 @@ func populateContainerHealth(app *spec.App) error {
 	return nil
 }
 
+func populateEnvFrom(app *spec.App) error {
+	// iterate on the containers so that we can extract envFrom
+	// that we have custom defined
+	for ci, c := range app.Containers {
+		for ei, e := range c.EnvFrom {
+			cmName := e.ConfigMapRef.Name
+
+			// we also need to check if the configMap specified if it exists
+			var cmFound bool
+			// to populate the envs we also need to know the data
+			// from configmaps defined in the app
+			for _, cm := range app.ConfigMaps {
+				// we will only populate the configMap that is specified
+				// not every configMap out there
+				if cm.Name != cmName {
+					continue
+				}
+				// start populating
+				for k := range cm.Data {
+					// here we are directly referring to the containers
+					// from app.PodSpec.Containers because that is where data
+					// is parsed into so populating that is more valid thing to do
+					app.PodSpec.Containers[ci].Env = append(app.PodSpec.Containers[ci].Env, api_v1.EnvVar{
```

Comment:
I was considering it, as this might be confusing and hard to notice if this happens. 
But then I tried that (define two env variables with the same name) with kubernetes and it just uses later value without any warnings or errors.

But i just found out another interesting thing regarding envFrom in 1.6.
Apparently, env wins over envFrom.

if you have something like this

```yaml
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  name: foo
spec:
  template:
    metadata:
      name: foo
      labels:
        app: foo
    spec:
      containers:
      - image: quay.io/tomkral/sleeper
        name: foo
        env:
        - name: foo
          value: valueFromEnv
        envFrom:
        - configMapRef:
            name: foo
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: foo
data:
  foo: fromConfigMap
```

Than the value of `foo` in container is `valueFromEnv`.

We should do the same thing. This should be really easy, as later value in env wins, all we have is add envs from `envFrom` to begining of `env` list, not to the end.